media/css/firefox/new/trailhead/exp-thanks.scss

```diff
@@ -6,212 +6,129 @@ $font-path: '/media/fonts';
 $image-path: '/media/protocol/img';
 
 @import '../../../../protocol/css/includes/lib';
+@import '../../../../protocol/css/components/call-out';
 @import '../../../../protocol/css/components/notification-bar';
-@import '../../../../protocol/css/components/hero';
-
-// keep the button technically visible to grab correct download link for
-// backup download link
-.js #download-button-wrapper-desktop {
```

Comment:
I think you'll need to add these back when you add back the download button macro.